api/src/identity-access-management/domain/usecases/anonymize-gar-authentication-methods.usecase.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @typedef {function} updateGARAuhenticationMethods
+ * @param {Object} params
+ * @param {string} params.userIds
+ * @param {AuthenticationMethodRepository} params.authenticationMethodRepository
+ * @return {Promise<void>}
+ */
+export const anonymizeGARAuthenticationMethods = async function ({ userIds, authenticationMethodRepository }) {
```

Comment:
Pour respecter les standards de nommage, utiliser plutôt le nommage suivant :

```suggestion
export const anonymizeGarAuthenticationMethods = async function ({ userIds, authenticationMethodRepository }) {
```